### PR TITLE
More misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Typos both in source and comments
Found via `codespell -q 3 -I ../librecad-whitelist.txt --skip="*.ts,./libraries"`

### CODE REVIEW

File: libraries/libdxfrw/src/drw_header.cpp [hunk 1]
```diff
@@ -1526,7 +1526,7 @@ void DRW_Header::write(dxfWriter *writer, DRW::Version ver){
         else
             writer->writeDouble(40, 50.0);
         writer->writeString(9, "$CAMERAHEIGHT");
-        if (getDouble("$CAMERAHEIGTH", &varDouble))
+        if (getDouble("$CAMERAHEIGHT", &varDouble))
```
Comment (luzpaz) [inline libraries/libdxfrw/src/drw_header.cpp]: please review

**Comment by luzpaz** [comment]
please review

**Comment by luzpaz** [comment]
shall i squash the commits ?

**Comment by lordofbikes** [comment]
I assume that everything is OK with your PR, but I was busy on [Google Code-in](http://codein.withgoogle.com), which is nearly over now. I expect to handle the PR this week.

File: libraries/libdxfrw/src/drw_header.cpp (hunk 1 above)
Comment (luzpaz) [reply libraries/libdxfrw/src/drw_header.cpp]: @lordofbikes this is OK, right?

**Comment by lordofbikes** [comment]
Many thanks, everything looks fine!

**Pull request merged**